File: review/java_01/src/main/java/com/example/java_01/MainActivity.java
package com.example.java_01;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.RadioGroup;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {


    Button btnNew;
    RadioGroup radioGroup;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btnNew = findViewById(R.id.btnNew);

        radioGroup = findViewById(R.id.group);


        btnNew.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {

                int id = radioGroup.getCheckedRadioButtonId();

                if (id == R.id.Second) {
                    Intent intent = new Intent(getApplicationContext(), Main2Activity.class);
                    startActivity(intent);
                } else if (id == R.id.Third) {
                    Intent intent = new Intent(getApplicationContext(), Main3Activity.class);
                    startActivity(intent);
                }
                else {
                    Toast.makeText(MainActivity.this, "액티비티 선택", Toast.LENGTH_SHORT).show();

                }
            }
        });

    }
}

File: Chapter10/ex01_practice10-1/src/main/java/com/example/ex01_practice10_1/MainActivity.kt
package com.example.ex01_practice10_1

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.RadioButton
import android.widget.Toast
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.activity_sccond.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?)
    {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnMain.setOnClickListener {
            var id: Int = RadioGroup.checkedRadioButtonId
            if(id != -1)
            {
                val radio: RadioButton = findViewById(id)
                Toast.makeText(applicationContext,"On button click : ${radio.text}",
                    Toast.LENGTH_SHORT).show()

                if (radio == Sccond){
                    val intent = Intent(this@MainActivity, SccondActivity::class.java)
                    startActivity(intent)
                }
                else if (radio == Third){

                    val intent = Intent(this@MainActivity, T2::class.java)
                    startActivity(intent)
                }
            }
            else {
                Toast.makeText(this@MainActivity, "선택바람", Toast.LENGTH_SHORT).show()
            }



        }
    }
}
